tasks/eva_2287/correct_contig_with_chr_removed.py

```diff
@@ -0,0 +1,54 @@
+import hashlib
+import argparse
+import pymongo
+import traceback
+import logging
+
+
+def get_SHA1(variant_rec):
+    """Calculate the SHA1 digest from the seq, study, contig, start, ref, and alt attributes of the variant"""
+    h = hashlib.sha1()
+    keys = ['seq', 'study', 'contig', 'start', 'ref', 'alt']
+    h.update('_'.join([str(variant_rec[key]) for key in keys]).encode())
+    return h.hexdigest().upper()
+
+
+def correct(mongo_user, mongo_password, mongo_host, mongo_database='eva_accession_sharded'):
+    with pymongo.MongoClient(mongo_host, username=mongo_user, password=mongo_password) as mongo_handle:
```

Comment:
Please use method from pyutils to get handle without specifying the password in command line.

Reply:
Done. 
I was not using pyutils because I didn't want the dependency in this script that is intended to run just once but I agree passing the password in the command line is no a good practise. Tried using `get_mongo_uri_for_eva_profile` that would simplify things in this case but I wrote a test that uses a local mongo without authentication but that is not correctly handled by the functions in pyutils. Probably something we should fix in that library.

Reply:
you could potentially patch the `get_mongo_uri_for_eva_profile` method in the tests to return a uri for your local mongo instance?  I don't think it matters much for this script but just a thought.

Reply:
Thanks! Change done